keyboards/handwired/polykybd/lang/_gen_script_cols.py

```
@@ -0,0 +1,202 @@
+        fonts_yaml.append({
+            "lang":langname,"key":key,"block":(0x900+off,0x900+off+0x7f),
+            "pua_base":pua_base,"seq":", ".join(seq_parts)})
+
+# ============================================================================
+# Thai (Kedmanee, from xkb symbols/th)
+# ============================================================================
+# keysym name -> codepoint, parsed from keysymdef.h (Thai_*), plus ASCII helpers
+THAI_KS={}
+for line in open("/usr/include/X11/keysymdef.h"):
+    m=re.search(r'#define XK_(Thai_\w+)\s+0x[0-9a-fA-F]+\s+/\*\s*U\+([0-9A-Fa-f]{4})', line)
+    if m: THAI_KS[m.group(1)]=int(m.group(2),16)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)